### ISSUE TITLE
Please Mirror this repo on D.O.

### ISSUE THREAD

**Issue by saltednut** [ericduran/queues#15]
http://git.drupal.org/project/queues.git can be added as a mirror.  Then you just periodically do a push for that remote to keep it up to date and continue happily developing on github.

Why?  So distributions on D.O. can package your module!

**Comment by likewhoa** [comment]
Nothing prevents distributions from packaging this repo in their drush make files.

```
projects[queues][type] = module
projects[queues][download][type] = git
projects[queues][download][branch] = 7.x-1.x
projects[queues][download][url] = https://github.com/ericduran/queues.git
```

is all that you really need, assuming you want the 7.x-1.x branch. Perhaps using hubdrop.io can help automate this process.

**Comment by saltednut** [comment]
Yes that technically works, but it not pass the packaging whitelist because it only allows modules from drupal.org -- so if I want a packaged distro, the modules have to come from D.O.  See: https://www.drupal.org/developing/distributions/drupalorg

**Comment by likewhoa** [comment]
understood :)
